docs/install.md

```diff
@@ -122,13 +129,14 @@ This is borrowed from the [NixOS manual](https://nixos.org/nixos/manual/index.ht
 1. Obtain latest NixOS. For example:
 
 	```
-	wget https://releases.nixos.org/nixos/18.09/nixos-18.09.2257.235487585ed/nixos-graphical-18.09.2257.235487585ed-x86_64-linux.iso
+	wget https://releases.nixos.org/nixos/19.03/nixos-19.03.172361.cf3e277dd0b/nixos-minimal-19.03.172361.cf3e277dd0b-x86_64-linux.iso
 	```
+	Alternatively you can build NixOS from source by following the instructions at https://nixos.org/nixos/manual/index.html#sec-building-cd.
 
 2. Write NixOS iso to install media (USB/CD). For example:
 
 	```
-	dd if=nixos-graphical-18.09.2257.235487585ed-x86_64-linux.iso of=/dev/sdX
+	dd ifnixos-minimal-19.03.172361.cf3e277dd0b-x86_64-linux.iso
```

Comment:
`=` missing and why did you remove `of=/dev/sdX` ?

Reply:
fixed